src/odemis/driver/test/semnidaq_test.py

```diff
@@ -157,6 +155,14 @@ def setUpClass(cls) -> None:
         if sys.version_info < (3, 9):
             raise SkipTest("semnidaq does not work for Ubuntu 20.04 or lower")
 
+        try:
+            from odemis.driver import semnidaq
+            from odemis.driver.semnidaq import Acquirer
+        except ModuleNotFoundError as e:
+            raise SkipTest(
+                f"semnidaq driver not available: {e}. Install required packages for your Ubuntu version") from e
```

Comment:
it would be nice to add here which packages need to be installed. I think it's just `python3-nidaqmx` @pieleric ?